setup.cfg

```diff
@@ -1,7 +1,15 @@
 # flake8 does not support pyproject.toml, see:
 # https://github.com/PyCQA/flake8/issues/234
 [flake8]
-exclude = docs
+exclude = docs,venv
 max-line-length = 100
 
+[mypy]
+files = src,test
+ignore_missing_imports = true
+exclude = docs,build
 
+[tool:pytest]
+addopts = --cov=src/mypackage --cov-report html --cov-report term-missing -v
```

Comment:
Ref comment above. Does this config in `setup.cfg` generate a coverage report whenever you run pytest?

Reply:
Just saw the actual comment in the PR. I think we need to document this behaviour (as it is implicit).

Do you know if we can use this in `pyproject.toml`?